page-template-cluster.php

```diff
@@ -147,6 +139,50 @@
 	</section>
 	<?php endif; ?>
 	<!-- End Research -->
+	<!-- Start Academics -->
+	<?php if ( is_array( $cluster_programs ) && ! empty( $cluster_programs ) ) : ?>
+	<section class="jumbotron bg-inverse text-inverse" aria-labelledby="research-programs">
+		<div class="container">
+			<div class="row">
+				<div class="col-lg-8">
+					<h2 id="research-programs" class="h1 mb-4 text-primary font-weight-black section-heading"><?php echo $post->post_title; ?> Degree Programs</h2>
+					<?php if ( $cluster_prg_copy ) : ?>
+					<div class="mb-5">
+						<p><?php echo $cluster_prg_copy; ?></p>
+					</div>
+					<?php endif; ?>
+					<div class="mb-5">
+						<?php echo do_shortcode( '[ucf-degree-search placeholder="Search programs..."]' ); ?>
+					</div>
+				</div>
+				<div class="col-lg-1 hidden-md-down">
+					<hr class="hidden-xs hidden-sm hr-vertical hr-vertical-white center-block">
+				</div>
+				<div class="col-lg-3">
+					<h3 class="h5 mb-3"><span class="badge badge-inverse">Programs</span></h3>
+					<ul class="list-unstyled">
```

Comment:
Making this a bulleted list might help anchor each list item a little bit.  To avoid the excessive indent on the list, you could add `pl-3` to the `ul`.

Reply:
Updated